procedures/katello/release.md.erb

```diff
@@ -26,12 +26,13 @@
   * [ ] Set version to `<%= version %>`
   * [ ] Set prerelease to `<%= extra %>`
 <% unless is_rc %>  * [ ] Reset release to `1`
-<% end %>  * [ ] In each package dir, remove the old tarball, run `spectool -g *.spec` and `git annex add *.tar.gz` or `git annex add *.gem`
+<% end %>  * [ ] Fetch new source by running `obal source rubygem-katello`, then `git rm` the old source.
   * [ ] Edit `katello-repos/katello.repo` to set `gpgcheck=1`
```

Comment:
Also part of branching perhaps? In Foreman we also automatically do this via RPM macros to avoid manual work.

Reply:
I am documenting the *current* process ;)

Reply:
Still, we already have done so for all current released versions. Could also be moved to `procedures/katello/branch.md.erb`.